app/src/main/java/ar/com/wolox/android/example/ui/training/login/LoginPresenter.java

```diff
@@ -1,16 +1,27 @@
 package ar.com.wolox.android.example.ui.training.login;
 
+import android.support.annotation.Nullable;
 import android.util.Patterns;
+import java.util.List;
 import javax.inject.Inject;
+import ar.com.wolox.android.example.model.User;
+import ar.com.wolox.android.example.network.LoginService;
 import ar.com.wolox.wolmo.core.presenter.BasePresenter;
+import ar.com.wolox.wolmo.networking.retrofit.RetrofitServices;
+import ar.com.wolox.wolmo.networking.retrofit.callback.NetworkCallback;
+import okhttp3.ResponseBody;
 
 /**
  * LoginPresenter is used for user to Login
  */
 public class LoginPresenter extends BasePresenter<ILoginView> {
 
     @Inject
-    public LoginPresenter() {
+    RetrofitServices mRetrofitServices;
```

Comment:
If it is not going to be accesed from another place, make retrofitservices private